### PR TITLE
Update microsoft-defender-endpoint-android.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document with the license requirement (intune license)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8340

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md [hunk 1]
```diff
@@ -40,10 +40,11 @@ This topic describes how to install, configure, update, and use Defender for End
 ### Prerequisites
 
 - **For end users**:
+  - The end user to be assigned a Microsoft Intune license. For more information on how to assign licenses, see [Assign licenses to users](/azure/active-directory/users-groups-roles/licensing-groups-assign).
   - Microsoft Defender for Endpoint license assigned to the end user(s) of the app. See [Microsoft Defender for Endpoint licensing requirements](/microsoft-365/security/defender-endpoint/minimum-requirements#licensing-requirements)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-android.md]:
```suggestion
  - The end user must be assigned a Microsoft Intune license. For more information on how to assign licenses, see [Assign licenses to users](/azure/active-directory/users-groups-roles/licensing-groups-assign).
  - Microsoft Defender for Endpoint license assigned to the end user(s) of the app. See [Microsoft Defender for Endpoint licensing requirements](/microsoft-365/security/defender-endpoint/minimum-requirements#licensing-requirements).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. thanks

**Pull request merged**